src/SmartCommander/Models/OptionsModel.cs

```diff
@@ -48,11 +48,12 @@ static OptionsModel()
 
         public bool IsMaximized { get; set; }
 
-        public string LeftPanePath { get; set; } = "";
+        public string LeftPanePath { get; set; } = string.Empty;
 
-        public string RightPanePath { get; set; } = "";
+        public string RightPanePath { get; set; } = string.Empty;
 
         public bool IsDarkThemeEnabled { get; set; }
+        public bool AllowOnlyOneInstance { get; set; }
```

Comment:
Would prefer one instance to be the behavior by default.

Reply:
fixed